wbstack/sync/02-docker-fetch.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
 
 # Simply runs the fetch.sh script in a docker container
-docker run --rm -it -u $(id -u ${USER}):$(id -g ${USER}) -v $PWD:/tmp --entrypoint sh ghcr.io/wbstack/docker-build-util:latest wbstack/sync/02-fetch.sh
```

Comment:
I'm not sure what the real impact of this no longer being on a interactive TTY is? What's the benefit/trade off?

Reply:
Github actions can't run this if `-it` is used, as it is not an interactive TTY

https://github.com/wbstack/mediawiki/runs/4594221056?check_suite_focus=true